templates/docs/examples/patterns/badge/colors.html

```diff
@@ -0,0 +1,29 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Badge / Colors{% endblock %}
+
+{% block standalone_css %}patterns_badge{% endblock %}
+
+{% block content %}
+
+<span class="p-badge">9</span>
+
```

Comment:
Could we remove the empty lines, it makes embedded examples unnecessarily bloated:

<img width="576" alt="image" src="https://user-images.githubusercontent.com/83575/175301957-faaf8e90-9639-4e3a-b0a6-1b87a5c3ffbc.png">

Reply:
Done :+1: